src/rlang/vec.h

```diff
@@ -47,12 +47,12 @@ const void* r_raw_cbegin(r_obj* x) {
 }
 static inline
 r_obj* const * r_chr_cbegin(r_obj* x) {
-  return (r_obj* const *) STRING_PTR(x);
+  return (r_obj* const *) STRING_PTR_RO(x);
```

Comment:
The cast is no longer necessary I believe?

Reply:
Oh, thanks for catching!